developer-docs-site/docs/guides/building-from-source.md

```diff
@@ -0,0 +1,177 @@
+---
+title: "Building Aptos From Source"
+slug: "build-from-source"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Building Aptos From Source
+
+[Binary releases are available](../cli-tools/aptos-cli-tool/index.md), but if you want to build from source or develop on the Aptos tools, this is how.
```

Comment:
I would mention `CLI Binary releases are available`, as we don't build for `aptos-node` (though maybe we should?)